store/tikv/region_cache.go

```diff
@@ -196,17 +196,21 @@ func (c *RegionCache) dropRegionFromCache(verID RegionVerID) {
 }
 
 // loadRegion loads region from pd client, and picks the first peer as leader.
-func (c *RegionCache) loadRegion(key []byte) (*Region, error) {
-	var region *Region
-	var backoffErr error
-	for backoff := pdBackoff(); backoffErr == nil; backoffErr = backoff() {
+func (c *RegionCache) loadRegion(bo *Backoffer, key []byte) (*Region, error) {
```

Comment:
Add an error declaration to reduce the invoking of  `Backoff` and error judgments.
```
if err！= nil {
   if err =  bo.Backoff(boPDRPC, err); err != nil {
       return nil, errors.Trace(err)
   }
}
```